admin/release.py

```diff
@@ -152,6 +152,8 @@ def publish_docs(flocker_version, doc_version, environment):
     :param bytes flocker_version: The version of flocker to publish the
         documentation for.
     :param bytes doc_version: The version to publish the documentation as.
+        Attempting to publish documentation to a staging environment as a
+        documentation version publishes it as the version being updated.
```

Comment:
This explanation is a bit confusing. Make it more clear what the difference between `flocker_version` and `doc_version` is, both in production and staging.   A longer explanation in the body of the docstring (after the initial line, before the parameter list) would be useful.